app/src/main/java/com/example/ithaca_transit_android_v2/views/RouteCardCompact.kt

```diff
@@ -0,0 +1,180 @@
+package com.example.ithaca_transit_android_v2.views
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.DisplayMetrics
+import android.util.Log
+import android.view.View
+import android.widget.*
+import androidx.annotation.NonNull
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.example.ithaca_transit_android_v2.NetworkUtils
+import com.example.ithaca_transit_android_v2.R
+import com.example.ithaca_transit_android_v2.models.Coordinate
+import com.example.ithaca_transit_android_v2.models.Route
+import com.example.ithaca_transit_android_v2.presenters.RouteCardPresenter
+import com.example.ithaca_transit_android_v2.states.RouteCardState
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import io.reactivex.disposables.Disposable
+import kotlinx.android.synthetic.main.route_card_compact.*
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.async
+import kotlinx.coroutines.runBlocking
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.util.concurrent.locks.ReentrantLock
+import kotlin.collections.ArrayList
+
+class RouteCardCompact : AppCompatActivity(){
+
+    val c = this
+
+    var rvAdapter = RvAdapter(ArrayList())
+    private lateinit var searchDisposable: Disposable
+
+    private var subscribe: Disposable? = null
+    var dataList = ArrayList<Route>()
+
+
+    val end = Coordinate(42.45352923315714,-76.4802676615646)
+    val uid = "E4A0256E-5865-4E9F-8A5A-33747CAC7EBF"
+    val time = 1580887917.0
+    val destinationName = "Bill & Melinda Gates Hall"
+    val start = Coordinate(42.447319055865485,-76.48542509781194)
+    val arriveBy = false
+
+
+
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.route_card_compact)
+
+
+
+        val fab = findViewById<RelativeLayout>(R.id.fab)
+        val routeOptionsText = findViewById<TextView>(R.id.optionsText)
+
+
+
+        runBlocking {
+            val loc1 = CoroutineScope(Dispatchers.IO).async {
+                NetworkUtils().getSearchedLocations("Jameson")
+            }.await()
+
+            Log.v("loc1", loc1.toString())
+
+
+            val loc2 = CoroutineScope(Dispatchers.IO).async {
+                NetworkUtils().getSearchedLocations("Statler")
+            }.await()
+
+
+            Log.v("loc2", loc2.toString())
```

Comment:
this is for debugging, or does `loc1` and `loc2` represent start & end locations? if it's the former, please delete or leave a comment stating that this is temp. if it's the latter, please rename the variables